hack/get-kubeconfig.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# This script will print to stdout the kubeconfig for a ClusterDeployment.
+#
+
+set -e
+
+usage(){
+	echo "Usage: $0 [CLUSTERDEPLOYMENT_NAMESPACE/]CLUSTERDEPLOYMENT_NAME"
+	exit 1
+}
+
+
+# Get the name and namespace for the clusterdeployment to duplicate.
+CD_NAME_ARG=${1?missing ClusterDeployment name}
+CD_NAME_ARR=(${CD_NAME_ARG//// })
```

Comment:
how does this work? i'm not great at bash

Reply:
* `{string/substring/replacement}` replaces the first occurrence of *substring* with *replacement*
* `(string1 string2 string3)` creates an array with the elements in the list.

So, `(${CD_NAME_ARG//// })` replaces the first forward slash `/` in `CD_NAME_ARG` with a space then creates an array where the first element is everything before the new space and the second element is everything after the new space.